nncf/pruning/filter_pruning/algo.py

```diff
@@ -472,10 +514,14 @@ def _set_binary_masks_for_pruned_modules_globally_by_flops_target(self,
             cumulative_filters_importance = torch.zeros(filters_num[0]).to(device)
             # Calculate cumulative importance for all filters in this group
             for minfo in cluster.nodes:
-                normalized_weight = self.weights_normalizer(minfo.module.weight)
-                filters_importance = self.filter_importance(normalized_weight,
+                # TODO: check why here normalization is needed (in LEGR case normalization shouldn't be done)
+                # normalized_weight = self.weights_normalizer(minfo.module.weight)
```

Comment:
Please make sure that weights are not normalized prior to applying the ranking coefficients in the original work. I'd suggest doing several experiments with normalization, since that would align the search intervals for all coefficients.

Reply:
Checked, in LeGR paper and code weights used without normalization.

Experiments with/without normalization will be conducted to compare.